server/webhook_jira.go

```diff
@@ -28,6 +69,37 @@ type JiraWebhook struct {
 	IssueEventTypeName string `json:"issue_event_type_name"`
 }
 
+// toEventEnums converts a JiraWebhook struct into a slice of internal event identifiers
+func (jwh *JiraWebhook) toEventEnums() []string {
```

Comment:
toEventEnums() is where Jira's payload is converted into an array of associated enums that are used for the choices on the UI. The function is called when a webhook event comes in from Jira. Once the webhook event is converted into an array of enums, each subscription is checked to see if any one of the enums is relevant to what the user has configured in their subscription.